### PR TITLE
SubscriptionId, subname and subcontext renamed to organization

### PR DESCRIPTION
**Pull request by arvindsingh-cloud**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Abstracts/ServicesSecurityStatus.ps1 [hunk 1]
```diff
@@ -194,20 +194,20 @@ class ServicesSecurityStatus: ADOSVTCommandBase
...
 							. $extensionSVTClassFilePath
-							$svtObject = New-Object -TypeName $extensionSVTClassName -ArgumentList $this.SubscriptionContext.SubscriptionId, $_
+							$svtObject = New-Object -TypeName $extensionSVTClassName -ArgumentList $this.OrganizationContext.OrganizationId, $_
 						}
 					}
 					else 
 					{
                        # Extended type is already loaded. Create an instance of that type.
-					   $svtObject = New-Object -TypeName $extensionSVTClassName -ArgumentList $this.SubscriptionContext.SubscriptionId, $_						
+					   $svtObject = New-Object -TypeName $extensionSVTClassName -ArgumentList $this.OrganizationContext.OrganizationId, $_
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/Abstracts/ServicesSecurityStatus.ps1:210]: Rename all organizationId to organizationName

File: src/AzSK.ADO/Framework/Helpers/ContextHelper.ps1 [hunk 2]
```diff
@@ -169,6 +169,11 @@ class ContextHelper {
         $contextObj.Subscription = [Subscription]::new()
         $contextObj.Subscription.Id = [ContextHelper]::orgName
         $contextObj.Subscription.Name = [ContextHelper]::orgName
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/Helpers/ContextHelper.ps1:171]: won't this break?

File: src/AzSK.ADO/Framework/Listeners/RemoteReports/AIOrgTelemetry.ps1 [hunk 3]
```diff
@@ -140,7 +140,7 @@ class AIOrgTelemetry: ListenerBase {
 			"ScanKind" = [RemoteReportHelper]::GetSubscriptionScanKind(
 				$this.InvocationContext.MyCommand.Name,
 				$this.InvocationContext.BoundParameters);
-			"SubscriptionMetadata" = [JsonHelper]::ConvertToJsonCustomCompressed($SVTEventContextFirst.SubscriptionContext.SubscriptionMetadata);
+			"OrganizationMetadata" = [JsonHelper]::ConvertToJsonCustomCompressed($SVTEventContextFirst.OrganizationContext.OrganizationMetadata);
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/Listeners/RemoteReports/AIOrgTelemetry.ps1:143]: where is this metadata coming from?

File: src/AzSK.ADO/Framework/Models/Common/Context.ps1 [hunk 4]
```diff
@@ -3,6 +3,7 @@ class Context
 {
     [Account] $Account = [Account]::new();
     [Subscription] $Subscription = [Subscription]::new();
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/Models/Common/Context.ps1:5]: why do we need to keep this?

File: src/AzSK.ADO/Framework/Helpers/ContextHelper.ps1 (hunk 2 above)
Comment (arvindsingh-cloud) [reply src/AzSK.ADO/Framework/Helpers/ContextHelper.ps1]: subscription class removed

File: src/AzSK.ADO/Framework/Models/Common/Context.ps1 (hunk 4 above)
Comment (arvindsingh-cloud) [reply src/AzSK.ADO/Framework/Models/Common/Context.ps1]: Removed this class

File: src/AzSK.ADO/Framework/Listeners/RemoteReports/AIOrgTelemetry.ps1 [hunk 5]
```diff
@@ -293,37 +293,37 @@ class AIOrgTelemetry: ListenerBase {
...
-                    $telemetryEvent.properties.Add([TelemetryKeys]::SubscriptionName, $azureContext.Subscription.Name)
+                    $telemetryEvent.properties.Add([TelemetryKeys]::OrganizationName, $organizationContext.Organization.Name)
                 }
                 catch {
 					# Eat the current exception which typically happens when the property already exist in the object and try to add the same property again
 					# No need to break execution
 				}
                 try {
-                    $telemetryEvent.properties.Add("AzureEnv", $azureContext.Environment.Name)
+                    $telemetryEvent.properties.Add("AzureEnv", $organizationContext.Environment.Name)
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/Listeners/RemoteReports/AIOrgTelemetry.ps1:312]: ADOEnv

File: src/AzSK.ADO/Framework/Listeners/UserReports/README.txt [hunk 6]
```diff
@@ -22,15 +22,15 @@ By default, cmdlets open this folder upon completion of the cmdlet (we assume yo
...
 
 
  	\AttestationReport-<timestamp>.csv			
-	[This is the summary CSV file listing all applicable controls and their attestation details. This file will be generated only for the cmdlet Get-AzSKInfo -SubscriptionId <SubscriptionId> -InfoType AttestationInfo.]
+	[This is the summary CSV file listing all applicable controls and their attestation details. This file will be generated only for the cmdlet Get-AzSKInfo -OrganizationName <OrganizationName> -InfoType AttestationInfo.]
 
 
-  	\<Resource_Group_or_Subscription_Name_Folder>	
-	[This folder corresponds to the resource-group or subscription that was evaluated. If multiple resource groups were scanned, there is one folder for each resource group.]
+  	\<Resource_Group_or_Organization_Name_Folder>
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/Listeners/UserReports/README.txt:32]: Project_Or_

File: src/AzSK.ADO/Framework/Listeners/UserReports/README.txt [hunk 7]
```diff
@@ -22,15 +22,15 @@ By default, cmdlets open this folder upon completion of the cmdlet (we assume yo
...
 
  	\AttestationReport-<timestamp>.csv			
-	[This is the summary CSV file listing all applicable controls and their attestation details. This file will be generated only for the cmdlet Get-AzSKInfo -SubscriptionId <SubscriptionId> -InfoType AttestationInfo.]
+	[This is the summary CSV file listing all applicable controls and their attestation details. This file will be generated only for the cmdlet Get-AzSKInfo -OrganizationName <OrganizationName> -InfoType AttestationInfo.]
 
 
-  	\<Resource_Group_or_Subscription_Name_Folder>	
-	[This folder corresponds to the resource-group or subscription that was evaluated. If multiple resource groups were scanned, there is one folder for each resource group.]
+  	\<Resource_Group_or_Organization_Name_Folder>	
+	[This folder corresponds to the organization that was evaluated. If multiple resource groups were scanned, there is one folder for each resource group.]
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/Listeners/UserReports/README.txt:33]: or project

File: src/AzSK.ADO/Framework/Listeners/UserReports/README.txt [hunk 8]
```diff
@@ -45,7 +45,7 @@ The contents of the output folder are organized as under:
 		\EnvironmentDetails.LOG				
 		[This is the log file containing environment data of current PowerShell session.]
 		\SecurityEvaluationData.json		
-		[This is the detailed security data for each control that was evaluated. This file will be generated only for SVT cmdlets like Get-AzSKAzureServicesSecurityStatus, Get-AzSKSubscriptionSecurityStatus etc.]
+		[This is the detailed security data for each control that was evaluated. This file will be generated only for SVT cmdlets like Get-AzSKAADOSecurityStatus etc.]
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/Listeners/UserReports/README.txt:48]: remove an extra A

File: src/AzSK.ADO/Framework/Listeners/UserReports/README.txt [hunk 9]
```diff
@@ -61,5 +61,5 @@ You can use these outputs as follows -
   2) For 'Failed' or 'Verify' controls, look in the <resourceType>.LOG file (search for the text 'Failed' or by control-id) to help you understand why the control has failed.
   3) For 'Verify' controls, you will also find the SecurityEvaluationData.JSON file in the \Etc sub-folder handy. 
   4) For some controls, you can also use the 'Recommendation' field in the control output to quickly get to the PS command to address the issue.
-  5) Make changes as needed to the subscription/resource configs based on steps 2, 3 and 4. 
+  5) Make changes as needed to the organization/resources configs based on steps 2, 3 and 4.
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/Listeners/UserReports/README.txt:64]: organization/project

**Review by abhaydaga — APPROVED** [review]

**Pull request merged**